### PR TITLE
Add more FAQs for Timescale Cloud

### PR DESCRIPTION
**Pull request by CoolAssPuppy**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -78,6 +81,32 @@ and more. Use cases include complex monitoring and analytics; predicting the per
...
+Timescale Cloud manages all of the operational elements of your database so you can 
+focus on building your applications and not making sure the infrastructure works. We 
+ensure you have a secure, high availability environment where we manage the infrastructure 
+all the way down to setting up replications, point-in-time recovery, read replicas, 
+backups, and more.
+
+### **Which cloud providers and regions are supported by Timescale Cloud?** [](which-cloud-regions)
+Timescale Cloud is available in the following Clouds and Regions:
+
+* **Amazon Web Services (AWS)**: N. Virginia, Oregon, Ohio, N. California, São Paulo,
```
Comment (solugebefola) [inline faq.md]: I think we probably want the actual region codes here, rather than geographic names. cc: @mfreed

File: faq.md [hunk 2]
```diff
@@ -78,6 +81,32 @@ and more. Use cases include complex monitoring and analytics; predicting the per
...
+### **Why should I choose Timescale Cloud?** [](why-cloud)
+If you want all the benefits of TimescaleDB without the hassle of installing,
+maintaining, and administering the database itself, then Timescale Cloud is the
+best choice for you.
+
+Timescale Cloud manages all of the operational elements of your database so you can 
+focus on building your applications and not making sure the infrastructure works. We 
+ensure you have a secure, high availability environment where we manage the infrastructure 
+all the way down to setting up replications, point-in-time recovery, read replicas, 
+backups, and more.
```
Comment (solugebefola) [inline faq.md]: Please add 'Top' link to the end of all new FAQ entries (see other entries for example)

File: faq.md [hunk 3]
```diff
@@ -78,6 +81,32 @@ and more. Use cases include complex monitoring and analytics; predicting the per
...
+* **Amazon Web Services (AWS)**: N. Virginia, Oregon, Ohio, N. California, São Paulo, 
+Stockholm, Ireland, London, Paris, Frankfurt, Canada, Sydney, Singapore, Tokyo, 
+Mumbai, Seoul
+* **Microsoft Azure (Azure)**: East US, East US 2, West Europe, Southeast Asia
+* **Google Cloud Platform (GCP)**: Northern Virginia, Los Angeles, South Carolina, 
+Iowa, Oregon, São Paulo, Zürich, London, Frankfurt, Finland, Belgium, Netherlands, 
+Montréal, Sydney, Taiwan, Mumbai, Hong Kong, Tokyo, Singapore
+
+### **What is the difference between Timescale, TimescaleDB, and Timescale Cloud?** [](what-does-timescale-mean)
+Timescale is our company. We build a category-defining time-series database called TimescaleDB.
```
Comment (solugebefola) [inline faq.md]: Bold 'Timescale', 'TimescaleDB' (first instance), 'Timescale Cloud'?

File: faq.md [hunk 4]
```diff
@@ -78,6 +81,35 @@ and more. Use cases include complex monitoring and analytics; predicting the per
...
+Frankfurt (`eu-central-1`), Canada (`ca-central-1`), Singapore (`ap-southeast-1`), 
+Sydney (`ap-southeast-2`), Tokyo (`ap-northeast-1`), Mumbai (`ap-south-1`), Seoul (`ap-northeast-2`)
+* **Microsoft Azure (Azure)**: East US, East US 2, West Europe, Southeast Asia
+* **Google Cloud Platform (GCP)**: Northern Virginia, Los Angeles, South Carolina, 
+Iowa, Oregon, São Paulo, Zürich, London, Frankfurt, Finland, Belgium, Netherlands, 
+Montréal, Sydney, Taiwan, Mumbai, Hong Kong, Tokyo, Singapore
+[[Top]](#top)
+
+### **What is the difference between Timescale, TimescaleDB, and Timescale Cloud?** [](what-does-timescale-mean)
+**Timescale** is our company. We build a category-defining time-series database called **TimescaleDB**.
```
Comment (svenklemm) [inline faq.md]: not sure "our" is the right choice here since this usage would most likely exclude the reader

File: faq.md [hunk 5]
```diff
@@ -254,7 +288,7 @@ please [contact us][contact] and we'd be happy to discuss. [[Top]](#top)
 ### **Is TimescaleDB currently being used in production?** [](in-production)
 Yes. TimescaleDB is currently deployed in production across a variety of industries
 including manufacturing, energy, utilities, mining, oil and gas, finance, ad tech,
-smart spaces, and more.
+smart spaces, and more. [[Top]](#top)
```
Comment (solugebefola) [inline faq.md]: Nice catch

**Pull request merged**